iocore/net/quic/QUICAckFrameCreator.cc

```diff
@@ -114,7 +124,7 @@ QUICAckFrameCreator::_create_ack_frame(QUICEncryptionLevel level)
       ack_frame->ack_block_section()->add_ack_block({static_cast<uint8_t>(gap - 1), length - 1});
     } else {
       uint64_t delay = this->_calculate_delay(level);
-      ack_frame      = QUICFrameFactory::create_ack_frame(largest_ack_number, delay, length - 1);
+      ack_frame      = QUICFrameFactory::create_ack_frame(largest_ack_number, delay, length - 1, this->_frame_ids, this);
```

Comment:
I missed one thing on the previous PR. Name of `_frame_ids` seems to be confusing because it's a single number but not multiple numbers. Since it's a sequence number, how about `_latest_frame_id` ?

Also, I'd like to suggest adding `uint64_t QUICFrameGenerator::_issue_frame_id()` and incrementing the sequence number in it. It would reduce chance of recording frames with wrong ids.

```
QUICFrameFactory::create_ack_frame(largest_ack_number, delay, length - 1, this->_issue_frame_id(), this);
```

In generate_frame:
```
this->_records_frame(ack_frame.id(), info);
```